internal/util/duration.go

```diff
@@ -0,0 +1,105 @@
+package util
```

Comment:
I believe we have a util.Duration already if we were to make this change we should consider using that.